common/static/common/js/utils/clamp-html.test.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { clampHtmlByWords } from './clamp-html';
+
+let container;
+
+beforeEach(() => {
+  container = document.createElement("div");
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  document.body.removeChild(container);
+  container = null;
+});
+
+describe('ClampHtml', () => {
+  test.each([
```

Comment:
This is the DDT style testing you mentioned? I like it

Reply:
Yeah. Built into jest, which is nice.